Tests/NIOSSLTests/NIOSSLIntegrationTest.swift

```diff
@@ -1633,10 +1663,10 @@ class NIOSSLIntegrationTest: XCTestCase {
         do {
             try clientChannel.writeInbound(buffer)
             XCTFail("Did not error")
-        } catch NIOSSLError.shutdownFailed {
-            // expected
-        } catch {
-            XCTFail("Unexpected error: \(error)")
+        } catch  {
+            let errorCode = CNIOBoringSSL_ERR_get_error()
```

Comment:
Hmm, this is going a bit too far the other way. I was thinking more that we'd expect to just use `String(describing: error)`. That should give us a better error message (specifically, tell us which enum case we actually threw instead of lying to us about it being case 0).

Reply:
Thanks.  I dialed it in a bit to use `.sslError` and then validate the `.description` off of that error.